include/openssl/pem.h

```diff
@@ -489,6 +489,13 @@ OPENSSL_EXPORT EC_GROUP *PEM_read_bio_ECPKParameters(BIO *bio,
 OPENSSL_EXPORT int PEM_write_bio_ECPKParameters(BIO *out,
                                                 const EC_GROUP *group);
 
+// PEM_write_bio_PrivateKey_traditional calls |PEM_ASN1_write_bio| to write
+// out |x|'s private key in the "traditional" ASN1 format. Use
```

Comment:
what is the "traditional" ASN.1 format? is it specified in a particular RFC or PKCS document? does it impose restrictions on key algorithm/type?

Reply:
Not related to anything particular, they just used "traditional" as an alias for the ASN.1 encoding they were initially doing within the PEM files. They changed the original to use PKCS8 instead and created this API for the ASN1 encoding.
* https://github.com/openssl/openssl/commit/05dba8151bd418cdc111d62102aaf9f4e7bd2f3f